api/views/stories.py

```diff
@@ -0,0 +1,58 @@
+from api import app, db
+from api.models import Story, StoryPOI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request
+
+mod = Blueprint('stories', __name__)
+
+STORIES_URL = "/stories"
+STORIES_ID_URL = "/stories/<int:story_id>"
+
+@app.route(STORIES_URL, methods=['GET'])
+def get_stories():
+    if 'POI' in request.args:
+        stories = StoryPOI.query.filter(StoryPOI.poi_id == request.args.get('POI'))
+        if stories.count() == 0:
+            return create_response(status=404, message='No stories found for specified POI')
+        else:
+            return create_response({'stories': [s.to_dict() for s in stories]})
+    else:
+        stories = Story.query.all()
+        return create_response({'stories': [s.to_dict() for s in stories]})
+
+@app.route(STORIES_URL, methods=['POST'])
+def post_stories():
+    data = request.get_json()
+    new_story = row_constructor(Story, data)
+    db.session.add(new_story)
+    db.session.flush()
+

```

Comment:
this currently doesn't account for the case where `story_name` isn't sent in the request payload, and it's a required parameter. if it's not provided, you should return a useful error message to the client